recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/xen-chosen.dtsi

```diff
@@ -11,7 +11,7 @@
 
 / {
 	chosen {
-		bootargs = "dom0_mem=256M console=dtuart dtuart=serial0 dom0_max_vcpus=4 bootscrub=0 loglvl=all cpufreq=none sched=credit2 sched_ratelimit_us=100";
+		bootargs = "dom0_mem=256M console=dtuart dtuart=serial0 dom0_max_vcpus=4 bootscrub=0 loglvl=all cpufreq=none sched=credit2 sched_ratelimit_us=100 hmp-unsafe=true";
```

Comment:
Xen migration patch series doesn't contain CPUFreq stuff, so "cpufreq=none" is not needed I think

Reply:
Right you are! I looked at it but left it occasionally.

Reply:
BTW, it seems that credit2 is used by default (CONFIG_SCHED_CREDIT2_DEFAULT=y), so "sched=credit2" property can be dropped also.